src/pages/best-practices.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Best Practices for PWA Studio
+---
+
+# Best Practices for PWA Studio
+
+Here are some best practices to keep in mind when working with PWA Studio.
+
+## Dynamic Content
+
+Using dynamic content can be challenging due to shifting layouts. It is helpful to end users if the page layout is established as quickly as possible. Dynamic content can lead to page flow issues.
```

Comment:
Just a little clarification-- do you mean to establish the page layout as _early_ as possible since it can lead to flow page issues in the long run?

Reply:
Good distinction!